dotnetv3/S3/AbortMPUExample/AbortMPUExample/AbortMPU.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace AbortMPUExample
+{
+    using System;
+    using System.Threading.Tasks;
+    using Amazon.S3;
+    using Amazon.S3.Transfer;
+
+    /// <summary>
+    /// This example shows how to use the Amazon Simple Storage Service
+    /// (Amazon S3) to stop a multi-part upload process using the Amazon S3
+    /// TransferUtility. The example was created using the AWS SDK for .NET
+    /// version 3.7 and .NET Core 5.0.
+    /// </summary>
+    public class AbortMPU
+    {
+        public static async Task Main()
+        {
+            string bucketName = "doc-example-bucket";
+
+            // If the AWS region defined for your default user is different
```

Comment:
AWS Region on first use , Region thereafter. I won't repeat this comment.

Reply:
fixed